projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -460,7 +459,8 @@ export class IgxDropDownComponent extends IgxDropDownBaseDirective implements ID
     /**
      * @hidden @internal
      */
-    ngOnInit() {
+    // temp workaround until fix --> https://github.com/angular/angular/issues/34992
+    ngOnChanges() {
```

Comment:
you should not set the `id` on each `OnChanges` call. Check if the change is actually for the `id` and set it only then like this:
```
    ngOnChanges(changes: SimpleChanges) {
        if (changes.id) {
            this.toggleDirective.id = changes.id.currentValue;
        }
    }
```